pkgs/by-name/sc/scrutiny/package.nix

```diff
@@ -0,0 +1,71 @@
+{ buildNpmPackage
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+, lib
+, ...
+}:
+let
+  name = "scrutiny";
```

Comment:
```suggestion
  pname = "scrutiny";
```
since this is used as pname, we might as well name it like this

Reply:
Moved this to just statically set the pname in the scrutiny package